plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -1050,6 +1049,33 @@
 			}
 		}
 	}
+
+	.add-variation-container {
```

Comment:
I turned off stylelint when I edited this file to avoid this massive change. Ideally we would do that linter fix in a separate PR (or commit at the very least), but what's done is done! 😄

Reply:
There is a lot of duplication between this style and the `.add-attributes-container` style added in #37352. Perhaps it would make sense to refactor a bit to avoid this duplication.